library/src/main/java/com/schibsted/spain/barista/BaristaRule.java

```diff
@@ -0,0 +1,55 @@
+package com.schibsted.spain.barista;
+
+import android.app.Activity;
+import android.content.Intent;
+import com.schibsted.spain.barista.cleardata.ClearDatabaseRule;
+import com.schibsted.spain.barista.cleardata.ClearPreferencesRule;
+import com.schibsted.spain.barista.flakyespresso.FlakyActivityTestRule;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+public class BaristaRule<T extends Activity> implements TestRule {
+
+  private static final int DEFAULT_FLAKY_ATTEMPTS = 10;
```

Comment:
Let's stay on 10... All values are great. Increasing it adds reliability, and decreasing it gives fail-fast tests. But both are great. Let's move on with 10. We can change it if needed, so don't worry about this.